test/deal-investor/get-deal-investors-by-deal-identifier.api-test.ts

```diff
@@ -3,10 +3,11 @@ import { UkefId } from '@ukef/helpers';
 import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
 import { Api } from '@ukef-test/support/api';
 import { ENVIRONMENT_VARIABLES } from '@ukef-test/support/environment-variables';
-import { DealInvestorGenerator } from '@ukef-test/support/generator/deal-investor-generator';
 import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
 import nock from 'nock';
 
+import { GetDealInvestorGenerator } from '../support/generator/get-deal-investor-generator';
```

Comment:
Can we import from '@ukef-test` instead of from `..` please?

Reply:
Done